spinta/manifests/tabular/helpers.py

```diff
@@ -859,6 +859,46 @@ def leave(self) -> None:
         pass
 
 
+class UniqueReader(TabularReader):
+    type: str = 'unique'
+
+    def read(self, row: ManifestRow) -> None:
+        reader = self.state.stack[-1]
+
+        if not isinstance(reader, (
+            DatasetReader,
+            ResourceReader,
+            BaseReader,
+            ModelReader,
+            PropertyReader,
+        )):
+            return
+
+        if self.type not in reader.data:
+            reader.data['unique'] = {}
+
+        unique = reader.data['unique']
+
+        self.name = '_'.join(row[REF].split(', '))
+
+        unique[self.type] = {
```

Comment:
Unique values should be added on a name constructed from row[REF], for example if ref is `name, country`, then unique name should be `name_country`.

Reply:
I think it already is